### ISSUE TITLE
Changes to copyright info on files not displaying well in review view

### ISSUE THREAD

**Issue by fossygirl** [clearlydefined/website#293]
For this PR: 
https://github.com/clearlydefined/curated-data/pull/170 the link in the comment https://clearlydefined.io/curations/170 is showing a change to the license.

But the actual change is a change to a copyright statement in a file. The review view isn't making that clear to the user. We should find a better way to display this for reviewing.

**Comment by gianpaj** [comment]
Agreed on the file copyrights change. There should be something highlighting that there are changes in the files without having to expand to the Full Detail view when opening the modal.

Regarding the change in Declared license, the highlight is showing that because originally it didn't have a license.
https://clearlydefined.io/definitions/npm/npmjs/-/extract-opts/2.2.0

- Is that not the correct way to highlight the change in this situation?
- or should the website also take into account the Discovered license when highlighting or not the Declared license?

**Comment by gianpaj** [comment]
At the moment (in dev) the Copyright file changes are only visible inside the popover. So they're only visible after clicking a Copyright file _you know_ has been changed. We should probably fix this.

I made this curation in dev that has a Copyright file change to see this:
<img width="559" alt="clearlydefined" src="https://user-images.githubusercontent.com/899175/45362357-497b0900-b5dd-11e8-9aac-c22c9b63cfbe.png">
https://dev.clearlydefined.io/curations/193

I believe there's another bug here. The Attribution Copyright should be highlighted. Is that correct?

Maybe we can show that there's a change in the Copyrights for that file with a red dot or similar, once the Full Detail View/Modal is opened.

---

Regarding showing the Declared license change in the Component List, I'm not sure how to highlight this.
FYI at the moment we can't see the Declared license highlighted in Curation 170 because it has already been updated by 169:
https://github.com/clearlydefined/curated-data/pull/169
